resources/config/laravel-sitemap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+
+  // Settings for GuzzleHttp\Client
+  'cookies' => true,
```

Comment:
Instead of using string, use the guzzle constants (like `RequestOptions::COOKIES`) here. That will make it even more clear which options can be used and added here.

 I don't mind an import of the namespace on the top of the file.